modules/checkbox/css/README.md

```diff
@@ -37,4 +40,4 @@ made fully accessible.
 
 ## Labels
 
-Render a checkbox inside a [canvas-kit-form-field](../../form-field/css) to add a label.
+Render a select inside a [canvas-kit-form-field](../../form-field/css) to add a label.
```

Comment:
Was this change intentional? I think it is supposed to be "checkbox", right?

Reply:
yes